lib/ui/screens/top_users/top_users_screen.dart

```diff
@@ -0,0 +1,150 @@
+                                DataColumn(
+                                  label: setTextValue("Name", true),
+                                ),
+                                DataColumn(
+                                  label: setTextValue("Email", true),
+                                ),
+                                DataColumn(
+                                  label: setTextValue("Score", true),
+                                ),
+                              ],
+                              rows: userList.map((user) {
+                                return DataRow(
+                                  cells: [
+                                    DataCell(
+                                      setTextValue(user.displayName, false),
+                                    ),
+                                    DataCell(setTextValue(user.email, false)),
+                                    DataCell(
+                                      setTextValue(
+                                        user.score.toString(),
+                                        false,
+                                      ),
+                                    ),
+                                  ],
+                                );
+                              }).toList(),
+                              border: TableBorder.all(
+                                width: 0.5,
+                                color: Colors.white,
+                              ),
+                            );
+                          } else {
+                            return Text(appResponse.message);
+                          }
+                        }
+                      }
+                      return const SizedBox.shrink();
+                    },
+                  ),
+                ),
+              ),
+            ),
+          ),
+        );
+      },
+    );
+  }
+
+  Widget setHeadLines() {
+    return Container(
+      padding: const EdgeInsets.all(8),
+      decoration: BoxDecoration(
+        border: Border.all(color: Colors.white, width: 0.5),
+      ),
+      child: Row(
+        mainAxisSize: MainAxisSize.min,
+        mainAxisAlignment: MainAxisAlignment.spaceAround,
+        children: [
+          setTextValue("Name", true),
+          setTextValue("Email", true),
+          setTextValue("Score", true),
+        ],
+      ),
+    );
+  }
+
+  Widget setValues(User user) {
+    return Container(
+      padding: const EdgeInsets.all(8),
+      decoration: BoxDecoration(
+        border: Border.all(color: Colors.white, width: 0.5),
+      ),
+      child: Row(
+        mainAxisSize: MainAxisSize.min,
+        mainAxisAlignment: MainAxisAlignment.spaceAround,
+        children: [
+          setTextValue(user.displayName, false),
+          setTextValue(user.email, false),
+          setTextValue(user.score.toString(), false),
+        ],
+      ),
+    );
+  }
+
+  Widget setTextValue(String strName, bool columnHeader) {
```

Comment:
Extracting widgets to a method is considered as a Flutter anti-pattern

Reply:
Made it as a stateless widget class now.